src/Microsoft.VisualStudio.ProjectSystem.Managed/Threading/Tasks/TaskResult.cs

```diff
@@ -10,13 +10,13 @@ namespace Microsoft.VisualStudio.Threading
     internal static class TaskResult
```

Comment:
💭 Why not just remove this class?

Reply:
It crossed my mind but `TplExtensions` is a mouthful.

Reply:
🤪